app/services/score_engine.py

```
@@ -0,0 +1,131 @@
+        score += 15
+    
+    return min(score + score_conectores, 100)
+
+def evaluar_compromiso(texto: str) -> int:
+    """
+    Evalúa el compromiso basándose en:
+    - Expresiones de motivación
+    - Planes específicos
+    - Expresiones de dedicación
+    """
+    score = 0
```

Ruff (0.12.2)

Local variable `score` is assigned to but never used

Remove assignment to unused variable `score`

(F841)